src/test/java/cloud/fogbow/ras/core/OrderStateTransitionerTest.java

```diff
@@ -21,7 +22,7 @@
 import org.powermock.modules.junit4.PowerMockRunner;
 
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({SharedOrderHolders.class, DatabaseManager.class})
+@PrepareForTest({SharedOrderHolders.class, DatabaseManager.class, OrderStateTransitioner.class})
```

Comment:
Remember to check if these annotations are necessary because BaseUnitTests already have some of that.

Reply:
Okay, Let's go check that.